pkg/api/routes.go

```diff
@@ -96,7 +90,7 @@ func (rh *RouteHandler) SetupRoutes() {
 	rh.c.Router.PathPrefix("/swagger/v2/").Methods("GET").Handler(httpSwagger.WrapHandler)
 	// Zot Search Extension Router
```

Comment:
Remove this comment and say
// Setup routes for extensions based on configuration